clients/go/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"goclients/cacclient"
+	"goclients/expclient"
+)
+
+func main() {
+	// Example usage
+	tenant := "dev"
+	pollingFrequency := 1
+	cachostName := "http://localhost:8080"
+
+	client := cacclient.NewCacClient(tenant, pollingFrequency, cachostName)
+	client.CreateNewCacClient()
+	fmt.Println("Default Configs", client.GetCacConfig(nil, nil))
+
+	expClient := expclient.NewExperimentationClient(tenant, pollingFrequency, cachostName)
+	expClient.CreateNewExperimentationClient()
+	fmt.Println("Running experiments:", expClient.GetRunningExperiments())
```

Comment:
@namitgoel please add examples for fetching a config, resolving, etc. Basically one for each function in the client

Reply:
done